maintenance/scripts/refresh_admin.sh

```diff
@@ -15,7 +15,6 @@ npm rebuild node-sass
 cd /var/www/html/frontend/admin
 composer install
 php artisan config:clear
-npm run h2-build
```

Comment:
I think this script should still run an h2-build.  The first half of it already runs at the /frontend level so it could be there.

Reply:
Done in 3fa5c9b5